### DISCUSSION TITLE
Add option to import prompts from Fabric library

### DISCUSSION THREAD

**Discussion by 5183nischal** [karthink/gptel#271] [Ideas]
[Fabric](https://github.com/danielmiessler/fabric) is a project that provides easy access to crowd sourced tried and true prompts and I wonder if it might integrate well with gptel. Thanks!

**Comment by karthink** [comment]
@5183nischal Thanks. This looks interesting, but I don't understand exactly what it is.  A system for composing prompts? Combining them? Sharing them?

> The Fabric ecosystem has three primary components, all named within this textile theme.
>
>     The Mill is the (optional) server that makes Patterns available.
>     Patterns are the actual granular AI use cases (prompts).
>     Stitches are chained together Patterns that create advanced functionality (see below).
>     Looms are the client-side apps that call a specific Pattern hosted by a Mill.

I'm afraid I don't have the time right now to devote to understanding or experimenting with these abstractions.

In any case, if you want to use the "patterns" provided by fabric with gptel and would like to write some glue code or a gptel extension, I can help.  But it's unlikely I'll be able to work on integrating it into the core package.

**Comment by braineo** [comment]
it seems can be as simple as just using the prompts provided by this repo? 

https://github.com/danielmiessler/fabric?tab=readme-ov-file#just-use-the-patterns


gptel is already capable of downloading prompts from https://github.com/f/awesome-chatgpt-prompts, the project's patterns folder is full of system messages like https://github.com/danielmiessler/fabric/blob/main/patterns/create_summary/system.md
